huaweicloud/services/identitycenter/resource_huaweicloud_identitycenter_group.go

```diff
@@ -0,0 +1,260 @@
+		getIdentityCenterGroupHttpUrl = "v1/identity-stores/{identity_store_id}/groups/{group_id}"
+		getIdentityCenterGroupProduct = "identitystore"
+	)
+	getIdentityCenterGroupClient, err := cfg.NewServiceClient(getIdentityCenterGroupProduct, region)
+	if err != nil {
+		return diag.Errorf("error creating Identity Center Client: %s", err)
+	}
+
+	getIdentityCenterGroupPath := getIdentityCenterGroupClient.Endpoint + getIdentityCenterGroupHttpUrl
+	getIdentityCenterGroupPath = strings.ReplaceAll(getIdentityCenterGroupPath, "{identity_store_id}",
+		fmt.Sprintf("%v", d.Get("identity_store_id")))
+	getIdentityCenterGroupPath = strings.ReplaceAll(getIdentityCenterGroupPath, "{group_id}", d.Id())
+
+	getIdentityCenterGroupOpt := golangsdk.RequestOpts{
+		KeepResponseBody: true,
+	}
+	getIdentityCenterGroupResp, err := getIdentityCenterGroupClient.Request("GET", getIdentityCenterGroupPath,
+		&getIdentityCenterGroupOpt)
+
+	if err != nil {
+		return common.CheckDeletedDiag(d, err, "error retrieving Identity Center Group")
+	}
+
+	getIdentityCenterGroupRespBody, err := utils.FlattenResponse(getIdentityCenterGroupResp)
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	createAt := utils.PathSearch("created_at", getIdentityCenterGroupRespBody, 0).(float64)
+	updateAt := utils.PathSearch("updated_at", getIdentityCenterGroupRespBody, 0).(float64)
+	mErr = multierror.Append(
+		mErr,
+		d.Set("region", region),
+		d.Set("name", utils.PathSearch("display_name", getIdentityCenterGroupRespBody, nil)),
+		d.Set("description", utils.PathSearch("description", getIdentityCenterGroupRespBody, nil)),
+		d.Set("created_at", utils.FormatTimeStampRFC3339(int64(createAt)/1000, false)),
+		d.Set("updated_at", utils.FormatTimeStampRFC3339(int64(updateAt)/1000, false)),
+	)
+
+	return diag.FromErr(mErr.ErrorOrNil())
+}
+
+func resourceIdentityCenterGroupUpdate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	cfg := meta.(*config.Config)
+	region := cfg.GetRegion(d)
+
+	// updateIdentityCenterGroup: update Identity Center group
+	var (
+		updateIdentityCenterGroupHttpUrl = "v1/identity-stores/{identity_store_id}/groups/{group_id}"
+		updateIdentityCenterGroupProduct = "identitystore"
+	)
+	updateIdentityCenterGroupClient, err := cfg.NewServiceClient(updateIdentityCenterGroupProduct, region)
+	if err != nil {
+		return diag.Errorf("error creating Identity Center Client: %s", err)
+	}
+
+	updateIdentityCenterGroupPath := updateIdentityCenterGroupClient.Endpoint + updateIdentityCenterGroupHttpUrl
+	updateIdentityCenterGroupPath = strings.ReplaceAll(updateIdentityCenterGroupPath, "{identity_store_id}",
+		fmt.Sprintf("%v", d.Get("identity_store_id")))
+	updateIdentityCenterGroupPath = strings.ReplaceAll(updateIdentityCenterGroupPath, "{group_id}", d.Id())
+
+	updateIdentityCenterGroupOpt := golangsdk.RequestOpts{
+		KeepResponseBody: true,
+	}
+	updateIdentityCenterGroupOpt.JSONBody = utils.RemoveNil(buildUpdateIdentityCenterGroupBodyParams(d))
```

Comment:
if there is another uneatable parameter,  buildUpdateIdentityCenterGroupBodyParams maybe return `map[string]interface{}{"operations": []}` when description is not changed and the API will fail.

so I think it's better to add d.HasChange in the outside of `buildUpdateIdentityCenterGroupBodyParams `